makefile

```
@@ -0,0 +1,4 @@
+.PHONY: build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)